### DISCUSSION TITLE
0.5.37-alpha.1

### DISCUSSION THREAD

**Discussion by jfarmer08** [jfarmer08/homebridge-wyze-smart-home#157] [Show and tell]
## What's Changed
- Support for Siren
- Support for Garage Door
- Support for Spotlight

* Main by @jfarmer08 in https://github.com/jfarmer08/homebridge-wyze-smart-home/pull/154
* Main by @jfarmer08 in https://github.com/jfarmer08/homebridge-wyze-smart-home/pull/155
* Update wyze-api by @jfarmer08 in https://github.com/jfarmer08/homebridge-wyze-smart-home/pull/156

**Full Changelog**: https://github.com/jfarmer08/homebridge-wyze-smart-home/compare/v0.5.36...v0.5.37-alpha.1

<hr /><em>This discussion was created from the release <a href='https://github.com/jfarmer08/homebridge-wyze-smart-home/releases/tag/v0.5.37-alpha.1'>0.5.37-alpha.1</a>.</em>

**Comment by sisuwest** [comment]
I see this supports the garage door opener, but I don’t see it as an accessory in Home Bridge. What do I need to do to get it enabled?
